include/proj/crs.hpp

```diff
@@ -346,6 +358,9 @@ class BoundCRS : public CRS {
              const operation::TransformationNNPtr &transformationIn);
     INLINED_MAKE_SHARED
 
+    bool isTOWGS84Compatible() const;
+    std::string getVDatumPROJ4GRIDS() const;
```

Comment:
Is there a particular reason for using including the "PROJ4" name here? Why not  ``getVDatumPROJGRIDS`` or simply ``getVDatumGrids``?

Reply:
Yes, that's the name used by the GDAL WKT1 EXTENSION. The purpose here is to be compatible with the past.

```
VERT_CS["EGM2008 geoid height",
    VERT_DATUM["EGM2008 geoid",2005,
        EXTENSION["PROJ4_GRIDS","egm08_25.gtx"],
        AUTHORITY["EPSG","1027"]],
    UNIT["metre",1,
        AUTHORITY["EPSG","9001"]],
    AXIS["Up",UP],
    AUTHORITY["EPSG","3855"]]
```
The equivalent WKT2 will be
```
BOUNDCRS[
    SOURCECRS[
        VERTCRS["EGM2008 geoid height",
            VDATUM["EGM2008 geoid"],
            CS[vertical,1],
                AXIS["up",up,
                    LENGTHUNIT["metre",1]],
            ID["EPSG",3855]]],
    TARGETCRS[
        GEODCRS["WGS 84",
            DATUM["WGS_1984",
                ELLIPSOID["WGS 84",6378137,298.257223563,
                    LENGTHUNIT["metre",1]]],
            PRIMEM["Greenwich",0,
                ANGLEUNIT["degree",0.0174532925199433]],
            CS[ellipsoidal,3],
                AXIS["latitude",north,
                    ORDER[1],
                    ANGLEUNIT["degree",0.0174532925199433]],
                AXIS["longitude",east,
                    ORDER[2],
                    ANGLEUNIT["degree",0.0174532925199433]],
                AXIS["ellipsoidal height",up,
                    ORDER[3],
                    LENGTHUNIT["metre",1]],
            ID["EPSG",4979]]],
    ABRIDGEDTRANSFORMATION["EGM2008 geoid height height to WGS84 ellipsoidal height",
        METHOD["GravityRelatedHeight to Geographic3D"],
        PARAMETERFILE["Geoid (height correction) model file","egm08_25.gtx",
            ID["EPSG",8666]]]]
```

Reply:
Ah, okay. Is the function going to be used to create both WKT1 and WKT2 strings? Just wondering if we'll be stuck with a hardcoded version number in a (API?) function that will stay around for a long time.

Reply:
The function is mostly to re-create GDAL WKT1 string from the internal model (using BoundCRS approach internally). And I've also used it for now to regenerate PROJ strings using the +geoidgrids params, but that might change later to use pipeline and +proj=vgridshift